packages/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -341,7 +340,9 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
 
   @override
   @server
-  Future<bool> deleteExpiredKeys() async {
+  @client
+  Future<bool> deleteExpiredKeys({bool? optimizeCommits = false}) async {
```

Comment:
Why not call it 'skipCommit' same as the existing flag name?

Reply:
makes more sense, let me get that changed

Reply:
change made as part of [fefa7d4](https://github.com/atsign-foundation/at_server/pull/1713/commits/fefa7d41067053944799aff620786d1a84b19694)